### DISCUSSION TITLE
12 months and still going

### DISCUSSION THREAD

**Discussion by helo-head** [HomeSpan/HomeSpan#297] [Show and tell]
I now have two Homespan projects deployed on M5Stickc+ hardware that are both stable and reliable. 

The first is an automated garage door vent/fan that takes into consideration the position of the door (sonic sensor), as well as temperature and humidity  to determine if the vent should be open and how fast the fan should run. This project has been running flawlessly for 12 months. That's more then I can say for some commercial products. 

I just finished the second project which adds HomeKit/siri functionality to my garage door opener. I was not happy with the mfg's HomeKit  integration which relied on their cloud services, so I just built a solution based on a spare M5Stickc+, relay and spare remote. Granted I had to configure it as a simple switch in HomeKit but it works every time ... It does feel a bit weird having to tell Siri to "turn on garage door  opener" but  rule 25 says "never argue with success".

Keep up the great work ... It's very rare in life to find solutions that are reasonable easy to use, get the job done and are reliable.

**Comment by Xorlent** [comment]
100% agree.  This library resulted in the most useful, trouble-free coding experience I've ever had!

**Comment by HomeSpan** [comment]
Really glad you are both having a good experience with the library! @helo-head, you may want to consider using Siri Shortcuts and create a phrase like "open bay door" that triggers the "on" function.   Of course if this works then the problem is remembering what phrase you used for the action 😄

**Comment by helo-head** [comment]
Ahhh ... I had not thought of that. As to memory well your right that does seem to be an issue as I get older :)
